src/codegen.cpp

```diff
@@ -7159,10 +7159,11 @@ extern "C" void jl_init_codegen(void)
     initializeAnalysis(Registry);
 #endif
 
-    InitializeNativeTarget();
-    InitializeNativeTargetAsmPrinter();
-    InitializeNativeTargetAsmParser();
-    InitializeNativeTargetDisassembler();
+    InitializeAllTargets();
```

Comment:
Is there a way to call InitializeNVPTXTarget*() when Julia is being built with the NVPTX back-end ?